tools/Install/setup.sh

```diff
@@ -138,9 +138,9 @@ fi
 
 XMOS_TAG=$2
 
-shift 1
+shift 2
 
-OPTIONS=s:a:m:d:h:x
+OPTIONS=s:a:m:d:hx:
```

Comment:
Did you intend to join the h and x options without an intervening colon?

Reply:
The column was wrong in the original code. It should be added only when we are reading an extra CLI argument.
In this case '-h' doesn't have any extra argument, while '-x' does.

Reply:
Thanks for the clarification.